lib/host-sp-messages/Cargo.toml

```diff
@@ -9,6 +9,6 @@ fletcher = "0.3"
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 serde_repr = "0.1"
 
-hubpack = { git = "https://github.com/cbiffle/hubpack" }
+hubpack = { git = "https://github.com/cbiffle/hubpack", rev = "df08cc3a6e1f97381cd0472ae348e310f0119e25" }
```

Comment:
Why this hash as opposed to a branch? Is it what the `Cargo.lock` was on?

Reply:
This hash was actually used by one of the dependencies.  It is one commit before what was used before, but the diff is minimal. https://github.com/cbiffle/hubpack/commit/d98084b85cc45fe589a49dd95403d7b41533375c